### DISCUSSION TITLE
Changelog

### DISCUSSION THREAD

**Discussion by wodow** [motor-admin/motor-admin#90] [General]
Hi Pete & all,

Excellent tool! Thanks for releasing it. Have you considered adding a CHANGELOG to the repo? The regularity of updates is great, but it's currently not easy to follow what's changing. (unless I am stupid and have missed it).

CW / wodow

**Comment by omohokcoj** [comment]
@wodow thanks for the feedback. that's true - currently there are no any good ways to find out what has changed in the release.

But at this points Motor Admin is more like in a stable/maintenance stage when only minor improvements and bug fixes are released without any large new features.
While having a CHANGELOG file would be a nice addition - the time/cost of maintaining the file might be too large to justify it for those minor bugfix releases.

So maybe just using github releases with commits might be a way to find out what has been added/fixed in some particular motor admin version:
https://github.com/motor-admin/motor-admin-rails/compare/0.4.8...0.4.9
